derivation/src/test/scala/tofu/higherKind/derived/EmbedKSuite.scala

```diff
@@ -3,13 +3,52 @@ import cats.data.{IorT, NonEmptyChain}
 import cats.free.Free
 import derevo.derive
 import tofu.higherKind.Embed
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+import cats.data.OptionT
+import EmbedKSuite.Foo
+import tofu.syntax.monadic._
+import tofu.syntax.embed._
+import cats.instances.either._
+import cats.instances.option._
+import cats.syntax.traverse._
+import cats.syntax.either._
+import scala.util.Try
+
+class EmbedKSuite extends AnyFlatSpec with Matchers {
```

Comment:
It's strange that the class is named `EmbedKSuite`, while we don't have `EmbedK`